scripts/xrfi_run.py

```diff
@@ -13,6 +13,7 @@
 
 xrfi.xrfi_run(args.ocalfits_files, args.acalfits_files, args.model_files,
               args.data_files, a_priori_flag_yaml=args.a_priori_flag_yaml,
+              a_priori_ants_only=not(args.a_apriori_times_and_freqs),
```

Comment:
Looks like you need to add YAML flags to day threshold run script.

Reply:
I've added this. Thanks for catching!